src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -64,6 +92,15 @@ public void setNeutral() {
     leftMotorFollower.setControl(new NeutralOut());
   }
 
+  public void setVoltage(Voltage voltage) {
+    rightMotorLeader.setControl(voltageRequest.withOutput(voltage.in(Units.Volts)));
```

Comment:
We can just pass the volage directly, no need to convert
```suggestion
    rightMotorLeader.setControl(voltageRequest.withOutput(voltage));
```